packages/react/src/TreeView/TreeViewWithLeadingAction.stories.tsx

```diff
@@ -0,0 +1,88 @@
+import {GrabberIcon, IssueClosedIcon, IssueOpenedIcon} from '@primer/octicons-react'
+import type {Meta, StoryFn} from '@storybook/react'
+import React from 'react'
+import Box from '../Box'
+import Link from '../Link'
+import {Banner} from '../Banner'
+import {IconButton} from '../Button'
+import Octicon from '../Octicon'
+import {TreeView} from './TreeView'
+
+const meta: Meta = {
+  title: 'Private/Components/TreeViewWithLeadingAction',
+  component: TreeView,
+  decorators: [
+    Story => {
+      return (
+        <Box sx={{maxWidth: 400}}>
+          <Story />
+        </Box>
+      )
+    },
+  ],
+}
+
+export const LeadingAction: StoryFn = () => {
+  return (
+    <Box>
+      <Banner
+        title="High-risk feature"
+        description={
+          <p>
+            Usage of LeadingAction is discouraged due to high-accessibility risk. See
+            <Link inline={true} href="https://github.com/github/primer/issues/3446">
+              audit issue (staff only)
+            </Link>
```

Comment:
```suggestion
            Usage of LeadingAction is discouraged due to high-accessibility risk. See{' '}
            <Link inline={true} href="https://github.com/github/primer/issues/3446">
              audit issue (staff only)
            </Link>
```

Adds a space between the text/link